synapse/lib/stormlib/gen.py

```diff
@@ -105,10 +105,32 @@ class LibGen(s_stormtypes.Lib):
                        'desc': 'Type normalization will fail silently instead of raising an exception.'},
                   ),
                   'returns': {'type': 'node', 'desc': 'A lang:language node with the given code.'}}},
+        {'name': 'itAvScanResultByTarget',
+         'desc': 'Returns an it:av:scan:result node by deconflicting with a target and signature name, adding the node if it does not exist.',
+         'type': {'type': 'function', '_funcname': '_storm_query',
+                  'args': (
+                      {'name': 'type', 'type': 'str', 'desc': 'The target type.'},
+                      {'name': 'target', 'type': 'str', 'desc': 'The target value.'},
```

Comment:
Calling convention and docs need to be reconciled. Order not the same and there appear to be three required arguments?

```
function itAvScanResultByTarget(signame, type, targ, scanner=$lib.null, time=$lib.null, try=$lib.false)
```

vs

```
cli> storm $lib.print($lib.gen.itAvScanResultByTarget(type='someType', target='someTarget'))
Executing query at 2024/01/26 14:06:17.634
....
ERROR: ('StormRuntimeError', {'efile': 'ast.py', 'eline': 4758, 'mesg': 'itAvScanResultByTarget() missing required argument signame', 'highlight': {'hash': '09a9d89d4cc5ac22a976aeba5f80c07f', 'lines': (212, 212), 'columns': (40, 112), 'offsets': (6104, 6176)}})
```

Reply:
Fixed